README.rst

```diff
@@ -40,31 +40,27 @@ Next step is to install assets (js, css etc.):
     plupload
     ...
 
-Now you can follow the standard invenio installation and development procedures, such as running `inveniomanage collect`.
+Now you can follow the standard Invenio installation and development procedures, such as running `inveniomanage collect`.
 
 ---------------------
 Demosite installation
 ---------------------
-As INSPIRE uses `Mixer <https://github.com/klen/mixer>`_ to load database fixtures, you should install the library:
```

Comment:
Since it is part of the `setup.py` it should indeed come automatically when installing the overlay.

We could still have a note in the readme that we use mixer for pre-loading the database without the specific installation instruction.